Report.java

```diff
@@ -201,5 +204,83 @@ private void printSpriteCounts(PrintWriter printW, int i)
                 + sprites[j].getSoundBlocksForSprite());
             printW.println();
         }
-    }   
+    }
+
+    /**
+     * Print total variable usage counts.
+     *
+     * @param printW - the printWriter to use
+     * @param i - the submission to get count for
+     */
+    public void printVariableUsageForProgram(PrintWriter printW, int i)
+    {
+        String[] vars = submissions[i].getGlobalVariables();
+        if (vars.length > 0)
+        {
+            printW.println("\tGlobal Variables:");
+            for (int j = 0; j < vars.length; j++)
+            {
+                printW.println("\t\t" + vars[j]
+                    + " used "
+                    + submissions[i].getProgramVariableUsageCount(vars[j])
+                    + " time(s)");
+            }
+        }
+    }
+    
+    /**
+     * Print Variable name and usage count for Stage.
+     *
+     * @param printW - the printwriter to use
+     * @param i - the submission to get count for
+     */
+    public void printVariableUsageForStage(PrintWriter printW, int i)
+    {   
+        String[] vars = submissions[i].getGlobalVariables();
+        if (vars.length > 0)
+        {
+            printW.println("\tGlobal Variables:");
+            for (int j = 0; j < vars.length; j++)
+            {
+                printW.println("\t" + vars[j]
```

Comment:
I think this needs 2 tabs "\t\t" + vars[j] so it's inset from the Global Variables title?